internal/cephfs/controllerserver.go

```diff
@@ -800,6 +805,16 @@ func (cs *ControllerServer) CreateSnapshot(
 			}
 		}
 
+		// Update snapshot-name/snapshot-namespace/snapshotcontent-name details on
+		// subvolume snapshot as metadata in case snapshot already exist
+		metadata := k8s.GetSnapshotMetadata(req.GetParameters())
```

Comment:
get metadata only once and use it? we have two calls for `GetSnapshotMetadata` in createSnapshot